cmd/gateway/azure/gateway-azure.go

```diff
@@ -87,6 +91,7 @@ EXAMPLES:
      {{.Prompt}} {{.EnvVarSetCommand}} MINIO_ACCESS_KEY{{.AssignmentOperator}}azureaccountname
      {{.Prompt}} {{.EnvVarSetCommand}} MINIO_SECRET_KEY{{.AssignmentOperator}}azureaccountkey
      {{.Prompt}} {{.HelpName}} https://azureaccountname.blob.custom.azure.endpoint
+     {{.Prompt}} {{.EnvVarSetCommand}} AZURE_CHUNK_SIZE_MB {{.AssignmentOperator}}0.25
```

Comment:
We cannot add custom ENVs of this form either they have to prefixed with MINIO_ or not added at all.  Was adding a higher value explored here as default instead of 0.25?

Reply:
Sure, a higher value could work in our case. We could bump it to 1 MB, but as files can grow we and others may run into the same problem. Another thing I forgot to mention is you can only commit 50.000 chunks at once. Hence the possible size to upload to Azure is even lower.

Azure allows chunks of 100 MB and blobs of 4,75TB. Why limit the capabilities? 

Would you agree on changing the envvar to MINIO_AZURE_CHUNK_SIZE_MB?

Reply:
Also, the comment states there is a timeout of 30 seconds in utils.go. When the default chunksize is higher, could this result in timeouts for users with low bandwidth?

Reply:
> Also, the comment states there is a timeout of 30 seconds in utils.go. When the default chunksize is higher, could this result in timeouts for users with low bandwidth?

Yes, it may be, but those are always there and it is going to fit everything into one box for everyone. Azure gateway is generally designed to be deployed in the Azure infrastructure itself not necessarily outside of Azure - if it's outside it has limited use we are okay there.